src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const sortedMens = people.filter(({ sex, died }) => {
+    return century
+      ? sex === 'm' && Math.ceil(died / 100) === century
+      : sex === 'm';
+  });
+
+  const sumOfAge = sortedMens.reduce((sum, { born, died }) => {
```

Comment:
You have the same function for women. So you can make one function for both options.

Reply:
make sens, thanks